### PR TITLE
Mobile middle section

### PR DESCRIPTION
**Pull request by Gopxfs**
Request to merge the mobile middle section and about me to the main branch.
- Added about me section
- Added contact section
- Added projects cards
- Updated project screenshots

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,16 +28,148 @@
...
+          <ul class="about-me-list">
+          <li class="about-me-list-item">Codekit</li>
+          <li class="about-me-list-item">Github</li>
+          <li class="about-me-list-item">Codepen</li>
+          <li class="about-me-list-item">Gitlab</li>
+          <li class="about-me-list-item">Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </div>
```
Comment (smunozmo) [inline index.html]: - [x] Your portfolio is looking great! Let's make it even better by using `ul` and `li` tags also for the technology categories, e.g:

```
<ul>
    <li>Languages
        <ul>
            <li>JS</li>
            <li>HTML</li>
            <li>CSS</li>
        </ul>
    </li>    
    <li>Frameworks
        <ul>
            <li>Bootsrap</li>
            <li>RoR</li>
            <li>RSpec</li>
        </ul>
    </li>

     . . .   

</ul>
```

File: index.html [hunk 2]
```diff
@@ -28,16 +28,148 @@
...
+        <a href="https://github.com/Gopxfs" class="contact-github">
+          <img class="github-icon" src="/images/github-icon.png" alt="GitHub Logo">   
+        </a>
+        <a href="https://www.linkedin.com/in/gabriel-fonseca-sales-8bb64b236/" class="contact-linkedin">
+          <img class="linkedin-icon" src="/images/linkedin-logo.png" alt="LinkedIn Logo">
+        </a>
+        <a href="https://twitter.com/FonsecaSGabe" class="contact-twitter">
+          <img class="twitter-icon" src="/images/twitter-logo.png" alt="Twitter Logo">
+        </a>
+      </div>
```
Comment (smunozmo) [inline index.html]: - [x] Contact icons are almost ready! Let's use `ul` and `li` tags to list them as this is a better practice for listings in HTML.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Gopxfs, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
- Changed contact-icons from <_div_> to <_li_>
- Changed about-me headings from <_div_> to <_li_>
- Adjusted some spacements

**Review by m05e5 — APPROVED** [review]
# Hi @Gopxfs  , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[Highly Recommended]
- _( optional )_ Please try to fix the images in your project. most of them are broken 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
> # Hi @Gopxfs ,
> ### Status: Approved ✔️
> goodjob
> 
> 👍 You have done all the required changes asked by the previous reviewer. Well Done.
> 
> Your project is complete! There are some optional changes I've suggested to you to make your project best, other than that nothing else to say. Congratulations! 🎉
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> [Highly Recommended]
> 
> * _( optional )_ Please try to fix the images in your project. most of them are broken
> 
> **You may close this Pull Request and merge the changes**
> 
> ## **Keep being awesome devs and good luck with your next milestone!**
> Cheers and Happy coding!👏👏👏 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_m05e5_**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hey @m05e5 what do you mean by broken images?

**Pull request merged**